modules/s3/main.tf

```diff
@@ -0,0 +1,40 @@
+## Local Variables
+locals {
+  orca_bucket_names  = [for k, v in var.buckets : v.name if v.type == "orca"]
+}
+
+resource "aws_s3_bucket" "reports_bucket" {
```

Comment:
is there any requirement to enable versioning in this bucket?

Reply:
Nah. Different versions of the same report exist on different keys.